docs/sdkx-cocos2dx/notifications-android.mdx

```diff
@@ -0,0 +1,264 @@
+  width="full"
+/>{" "}
+
+Click on configure button and then click on the _push notifications_ option.
+
+Enter your FCM key credentials per app, via the
+**Settings page > App listing in the left navigation > Scroll down to Push
+Notifications** settings section for the app.
+
+<Image src="/static/books/helpshiftx/add_fcm_token.png" width="full" />{" "}
+
+For FCM users, the API key can be found at your
+[Firebase API Console](https://console.firebase.google.com/)
+
+<Image src="/static/books/android/fcm_console.png" width="full" />{" "}
+
+<br />
+
+### Configure SDK {#configure-sdkx-android}
+
+1. Send the device registration id to Helpshift via the `HelpshiftCocosBridge.registerDeviceToken` API
+
+```java
+public class MyFirebaseMessagingService extends FirebaseMessagingService {
+  // ...
+  @Override
+  public void onNewToken(String newToken) {
+    // your logic to save the token here for the app
+    HelpshiftCocosBridge.registerPushToken(newToken);
+  }
+}
+```
+
+2. Using FCM's new `FirebaseMessagingService` which provides "RemoteMessage" argument, you should use `HelpshiftCocosBridge.handlePush` API which takes a Map argument.
+
+```java
+@Override
+public void onMessageReceived(RemoteMessage message) {
+  Map<String, String> data = message.getData();
+  String origin = data.get("origin");
+  if (origin != null && origin.equals("helpshift")) {
+      HelpshiftCocosBridge.handlePushNotification(getApplicationContext(), data);
+  }
+}
+```
+
+## In-app notifications {#in-app-notifications}
+
+In-app notifications are similar to notifications in the [notification drawer](http://developer.android.com/guide/topics/ui/notifiers/notifications.html) . Unlike push notifications, they appear only when you app is running.
+
+These notifications are sent when an agent replies to a customer's issue. Your customers can go straight into the [conversation screen](/sdkx_android/support-tools#conversation-view) when they tap on the notification.
+
+<Centered >
+
+![](/static/books/sdkx_android/notification.png)
+
+</Centered>
+
+<Admonition type="info" title="Note">
+
+If the FCM device token is registered for push notifications, then in-app notifications will be disabled. In-app notifications are disabled to avoid duplicate notifications from both push notifications and in-app notifications.
+
+</Admonition>
+
+## Customizing notification icons {#notification-icon}
+
+By default the application icon is used as the notification icon. You can customize the notification icons using the config in the `install` call.
+
+```java
+import com.helpshift.HelpshiftCocosBridge;
+
+public class MyActivity extends Cocos2dxActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Add largeNotificationIcon config in install config map
+        Map<String, Object> config = new HashMap<>();
+        config.put("notificationIcon", "icon-image-file-name");
```

Comment:
pls check if we take file names or resources Ids here ?

Reply:
Ye we expect resources ids, Will fix